### PR TITLE
CLOUD-1045+CLOUD-1166: Add Transactional + AI Page Under `Solutions`

### PR DESCRIPTION
**Pull request by aishwaryaborkar**
# Notes for blog or docs authors:

-   The `image: ` field in frontmatter of blogs should always be a `png` (`svg` will not work for social previews)
-   You must also duplicate the asset used in `image :` inside of the [public folder](https://github.com/tembo-io/website/tree/main/public)
-   Please write a `description: ` in the frontmatter of any new blog or doc
-   Read more [here](https://github.com/tembo-io/website?tab=readme-ov-file#writing-a-blog-post-%EF%B8%8F)

### CODE REVIEW

**Comment by aishwaryaborkar** [comment]
Notes: 
- I am aware of the "jump" in spacing between Tembo AI and Tembo Transactional. I'm working on tightening up the spacing as Amy prefers the spacing in Tembo AI over Tembo Transactional, e.g. making the pages less long.

File: src/components/SolutionsTemplate.astro
Comment (aishwaryaborkar) [inline src/components/SolutionsTemplate.astro:1]: This file ended up being not needed so I will delete

File: src/components/JoinTemboCTA.astro [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			<div class='flex flex-col'>
+				<div
+					class='text-white font-bold font-secondary tracking-[0.47px] text-[24px] lg:text-[40px] leading-tight mb-3'
+				>
+					Join Tembo Today
+				</div>
+				<p class='text-lightGrey font-normal font-secondary text-sm'>
+					Support for all Postgres workloads—including Transactional,
+					Analytical, and AI
+				</p>
```
Comment (aishwaryaborkar) [inline src/components/JoinTemboCTA.astro:30]: Amy has mentioned she needs to give me updated copy for the CTA footer here.

**Review by sarahedkins — APPROVED** [review]
LGTM

File: src/components/SolutionsNav.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		<NavigationMenu.Root
+			orientation='vertical'
+			className='flex relative justify-start'
+		>
+			<NavigationMenu.List className='flex flex-column'>
+				<NavigationMenu.Item>
+					<NavigationMenu.Trigger className='flex flex-row gap-1 justify-between items-center'>
+						Solutions
+						{isOpen ? (
+							<img src='/arrow-up.svg' alt='minus symbol' />
```
Comment (DarrenBaldwin07) [inline src/components/SolutionsNav.tsx:19]: This chevron icon does look a little large 
Screenshot 2024-09-04 at 11 32 36 AM

File: src/components/SolutionsNav.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+						) : (
+							<img src='/arrow-down.svg' alt='minus symbol' />
+						)}
+					</NavigationMenu.Trigger>
+					{isOpen && (
+						<NavigationMenu.Content className='flex flex-column py-4 px-2 rounded-3xl bg-mwasi border border-otherGrey2 absolute top-8 left-0 w-[214px]'>
```
Comment (DarrenBaldwin07) [inline src/components/SolutionsNav.tsx:25]: The background looks transparent for me. Not super important but it does make the navigation options hard to see
Screenshot 2024-09-04 at 11 32 55 AM

File: src/pages/solutions/transactional.astro [hunk 4]
```diff
@@ -0,0 +1,156 @@
+---
```
Comment (DarrenBaldwin07) [inline src/pages/solutions/transactional.astro:1]: For me, all the images on `/transactional` look a little fuzzy (`/ai` looks fine though). Maybe they were not exported properly from figma?
Screenshot 2024-09-04 at 2 43 44 PM

Comment (aishwaryaborkar) [reply src/pages/solutions/transactional.astro:1]: I asked Mia to export them for me, I can ask her to try again with better quality

File: src/components/SolutionsNav.tsx (hunk 2 above)
Comment (aishwaryaborkar) [reply src/components/SolutionsNav.tsx:19]: That feels subjective to me. Did you have a specific dimension in mind?

Comment (DarrenBaldwin07) [reply src/components/SolutionsNav.tsx:19]: I was comparing to the design. Figma (the `nav updates` tab) shows 16x16 and preview uses 24x24. I do think 16x16 would look better

**Pull request merged**